### PR TITLE
uniform #9002

### LINKED ISSUE #9002: uniform

### PR DESCRIPTION
**Pull request by Namangarg110**
Close #9002

### CODE REVIEW

**Review by jkeane508 — CHANGES_REQUESTED** [review]
Hi,

Thanks for submitting a PR :)

As part of this task, you also need to write a unit test in the relevant file.

Thanks.

**Comment by Namangarg110** [comment]
> Hi,
> 
> Thanks for submitting a PR :)
> 
> As part of this task, you also need to write a unit test in the relevant file.
> 
> Thanks.

Hey @jkeane508,
I have added the unit test. However, it may require some improvements. Can you please review it and suggest changes??

**Review by jkeane508 — CHANGES_REQUESTED** [review]
Hey,

Thanks for submitting this PR,

Just a few questions about your implementation :)

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

File: ivy/functional/frontends/tensorflow/random.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import ivy
+from ivy.func_wrapper import with_supported_dtypes
+from ivy.func_wrapper import with_unsupported_dtypes
+from ivy.functional.frontends.tensorflow.func_wrapper import (
+    to_ivy_arrays_and_back,
+    to_ivy_dtype,
+)
+
+
+@with_supported_dtypes({"2.9.0 and below": ("float16", "bfloat16",
```
[the remark this answers is not in the document]
Comment (Namangarg110) [reply ivy/functional/frontends/tensorflow/random.py:10]: Hey sorry @jkeane508 ,
I missed your comment,

Actually, this function has specific datatypes that are unsupported, that is why I used 2 decorators i.e. @with_supported_dtypes and @with_unsupported_dtypes. Please let me know if it is wrong to do so 

Thanks & Best Regards,
Naman Garg

**Pull request closed without merging**